<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaskuDialogi</class>
 <widget class="QDialog" name="LaskuDialogi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1631</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lasku</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pic/pic.qrc">
    <normaloff>:/pic/lasku.png</normaloff>:/pic/lasku.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,2,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Asiakas</string>
         </property>
         <property name="buddy">
          <cstring>asiakas</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="AsiakasToimittajaValinta" name="asiakas" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Nimi ja
osoite</string>
         </property>
         <property name="buddy">
          <cstring>osoiteEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPlainTextEdit" name="osoiteEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Sähköposti</string>
         </property>
         <property name="buddy">
          <cstring>email</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="email"/>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="otsikkoEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Otsikko</string>
         </property>
         <property name="buddy">
          <cstring>otsikkoEdit</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Laskutus</string>
           </property>
           <property name="buddy">
            <cstring>laskutusCombo</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QComboBox" name="laskutusCombo"/>
           </item>
           <item>
            <widget class="QComboBox" name="maksuCombo"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="toimituspvmLabel">
           <property name="text">
            <string>Toi&amp;mituspäivä</string>
           </property>
           <property name="buddy">
            <cstring>toimitusDate</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="KpDateEdit" name="toimitusDate"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string> - </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KpDateEdit" name="jaksoDate"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="eraLabel">
           <property name="text">
            <string>Eräpäi&amp;vä</string>
           </property>
           <property name="buddy">
            <cstring>eraDate</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="KpDateEdit" name="eraDate"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Asiakkaan viite</string>
           </property>
           <property name="buddy">
            <cstring>asViiteEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="asViiteEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="kieliLabel">
           <property name="text">
            <string>&amp;Kieli</string>
           </property>
           <property name="buddy">
            <cstring>kieliCombo</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="kieliCombo"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="viivkorkoLabel">
           <property name="text">
            <string>Viivästyskorko</string>
           </property>
           <property name="buddy">
            <cstring>viivkorkoSpin</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="viivkorkoSpin">
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="viiteLabel">
           <property name="text">
            <string>Viite</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,2">
           <item>
            <widget class="QLabel" name="viiteText">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>00000</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="maksettuCheck">
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../pic/pic.qrc">:/pic/ok.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="infoLabel">
             <property name="text">
              <string>Lähetetty 12.2.2019</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../pic/pic.qrc">
        <normaloff>:/pic/Menu-Circles.png</normaloff>:/pic/Menu-Circles.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Lis&amp;ätiedot</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="lisatietoEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="ohjeNappi">
       <property name="text">
        <string>Ohje</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/ohje.png</normaloff>:/pic/ohje.png</iconset>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="luonnosNappi">
       <property name="text">
        <string>Tallenna luonnoksena</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/tiedostoon.png</normaloff>:/pic/tiedostoon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tallennaNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Tallenna lasku   F10</string>
       </property>
       <property name="text">
        <string>T&amp;allenna valmiina</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/tiedostoon.png</normaloff>:/pic/tiedostoon.png</iconset>
       </property>
       <property name="shortcut">
        <string>F12</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="esikatseluNappi">
       <property name="toolTip">
        <string>Näytä lasku  F10</string>
       </property>
       <property name="text">
        <string>Nä&amp;ytä</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/print.png</normaloff>:/pic/print.png</iconset>
       </property>
       <property name="shortcut">
        <string>F10</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="summaLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="valmisNappi">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTipDuration">
        <number>0</number>
       </property>
       <property name="text">
        <string>Tallenna ja tulosta</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/tulosta.png</normaloff>:/pic/tulosta.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="peruNappi">
       <property name="text">
        <string>Peru</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/peru.png</normaloff>:/pic/peru.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AsiakasToimittajaValinta</class>
   <extends>QWidget</extends>
   <header location="global">rekisteri/asiakastoimittajavalinta.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KpDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">tools/kpdateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>osoiteEdit</tabstop>
  <tabstop>email</tabstop>
  <tabstop>otsikkoEdit</tabstop>
  <tabstop>laskutusCombo</tabstop>
  <tabstop>maksuCombo</tabstop>
  <tabstop>toimitusDate</tabstop>
  <tabstop>jaksoDate</tabstop>
  <tabstop>eraDate</tabstop>
  <tabstop>asViiteEdit</tabstop>
  <tabstop>kieliCombo</tabstop>
  <tabstop>viivkorkoSpin</tabstop>
  <tabstop>lisatietoEdit</tabstop>
  <tabstop>ohjeNappi</tabstop>
  <tabstop>luonnosNappi</tabstop>
  <tabstop>tallennaNappi</tabstop>
  <tabstop>esikatseluNappi</tabstop>
  <tabstop>valmisNappi</tabstop>
  <tabstop>peruNappi</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>peruNappi</sender>
   <signal>clicked()</signal>
   <receiver>LaskuDialogi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1587</x>
     <y>808</y>
    </hint>
    <hint type="destinationlabel">
     <x>1629</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
